homework2/task_1.py

```diff
@@ -0,0 +1,4 @@
+# Напишите программу, которая принимает на вход вещественное число и показывает сумму его цифр.
+
+print('Сумма =', sum(
+    map(int, list(filter(lambda x: x.isdigit(), input('Введите число: '))))))
```

Comment:
Зачем же так усложнять, используя `filter` и `map` одновременно? Используйте только `map`:
```suggestion
print('Сумма =', sum(
    map(lambda x: int(x) if x.isdigit() else 0, input('Введите число: '))))
```